options/locale/locale_en-US.ini

```
@@ -1540,6 +1540,7 @@ issues.dependency.issue_closing_blockedby = Closing this issue is blocked by the
 issues.dependency.issue_close_blocks = This issue blocks closing of the following issues
 issues.dependency.pr_close_blocks = This pull request blocks closing of the following issues
 issues.dependency.issue_close_blocked = You need to close all issues blocking this issue before you can close it.
+issues.dependency.issue_batch_close_blocked = Cannot batch close issues that you choose, because issue (#%[1]d) still has open dependencies
```

Actually, we can't do so, it breaks the crowdin, the part after `#` will be treated as comment.

And it seems that the parentheses are not necessary.


```suggestion
issues.dependency.issue_batch_close_blocked = "Cannot batch close issues that you choose, because issue #%[1]d still has open dependencies"
```

---

Is it because of ini format?

---

> Is it because of ini format?

Yes, the `#` needs to be quoted (at the moment, only double-quote does the work)

---

Okay, maybe we should just refactor to always quote all values for consistency when there is no drawback.

---

But there is another case: ` key = there is quote "hello" inside `, or  ` key = "there is quote \"hello\" inside" `

I think people may like the first style.

IMO we need our ini package to parse and lint, and that the first step for  [Proposal] Use ICU message for i18n & l10n #23863

---

> What about backtick quote everywhere?

The backtick could only be parsed by the old ini package, it's not supported by Crowdin, that's why many translations are broken (some backticks are missing, then Gitea can't parse them correctly)

---

Hmm, could we for example convert to YAML? Or does Crowdin not support it?

---

I am not crowdin admin, and I am not sure whether it would cause whole re-translating.